src/System.Management.Automation/engine/SessionStateContainer.cs

```diff
@@ -2382,16 +2382,38 @@ private bool HasGetChildItemDynamicParameters(ProviderInfo providerInfo)
                         // Since the path contained glob characters or we are recursing and the
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path -Recurse -Name.Get-ChildItem -Path -Recurse -Name
```powershell
Cannot convert 'System.Object[]' to the type 'System.String' required by parameter 'Message'. Specified method is not supported.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\FileSystemProviderExtended.Tests.ps1: line 244
244:             Write-Warning $result

```